conf/jenkins_job.ini.example

```diff
@@ -0,0 +1,13 @@
+[jenkins]
+user=my_username
+password=my_secret_password
+url=https://my.jenkins.com/
+query_plugins_info=False
+
+[job_builder]
+ignore_cache=True
```

Comment:
We use this example config for running tests, so we disable cache on purpose here. I think we should treat it as example rather than default